File: nodes/string-contains/string-contains.process.py
async def process(inputs, settings, config, nodeConfig):
  text = str(inputs.get("text", ""))
  search = str(inputs.get("search", ""))
  
  if not settings.get("case_sensitive", True):
    text = text.lower()
    search = search.lower()
    
  position = text.find(search)
  return {
    "contains": position != -1,
    "position": position
  } 

File: sdks/nodejs/nodes/string-contains/string-contains.process.js
export default async ({inputs, settings, config}) => {

  const text = String(inputs.text || "");
  const search = String(inputs.search || "");
  
  if (!settings.case_sensitive) {
    const position = text.toLowerCase().indexOf(search.toLowerCase());
    return {
      contains: position !== -1,
      position
    };
  }
  
  const position = text.indexOf(search);
  return {
    contains: position !== -1,
    position
  };
}; 